test/test.js

```diff
@@ -7389,6 +7389,65 @@ exports['wrapCallback - default mapper discards all but first arg'] = function (
     });
 };
 
+exports.wrapAsync = function (test) {
+    var f = function (a, b) {
+        return new Promise(function (resolve) {
+            resolve(a + b);
+        });
+    };
+    _.wrapAsync(f)(1, 2).toArray(function (xs) {
+        test.same(xs, [3]);
+        test.done();
+    });
+};
+
+exports['wrapAsync - context'] = function (test) {
+    var o = {
```

Comment:
I don't think this test verifies that the context is correctly passed. You need to use `this` inside `f` in some way. This is probably a copy-paste error since the `wrapCallback` version checks the context with `test.equal(this, o);`.